### PR TITLE
ENH Use SearchableMultiDropdownField

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1618

CI requires https://github.com/silverstripe/silverstripe-framework/pull/11057 to be merged first

### CODE REVIEW

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]
LGTM, but needs a dependency bump for admin and framework.

**Comment by emteknetnz** [comment]
Have updated composer.json

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "phpunit/phpunit": "^9.6",
         "silverstripe/frameworktest": "^1",
         "squizlabs/php_codesniffer": "^3.7",
-        "silverstripe/cms": "^5",
+        "silverstripe/cms": "^5.2",
```
Comment (ChloeSartorelli) [inline composer.json]:
```suggestion
        "silverstripe/cms": "^5",
```
Update the framework and admin dependencies in `require`, not cms in `require-dev`

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Opps :-) Have updated

**Review by ChloeSartorelli — APPROVED** [review]
LGTM, will merge once admin and framework PRs are merged

**Pull request merged**